modules/Bio/EnsEMBL/Variation/TranscriptVariationAllele.pm

```diff
@@ -550,6 +550,198 @@ sub sift_score {
     return $self->{sift_score};
 }
 
+=head2 dbnsfp_cadd_prediction
+
+  Description: Return the qualitative CADD prediction for the effect of this allele.
```

Comment:
It is dangerously open to miss-interpretation that these predictions are not based on the transcript. We need a comment to this effect in these methods

Reply:
I will add a comment here later. But as far as I understand the score is not transcript based.